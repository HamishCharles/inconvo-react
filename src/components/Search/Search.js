import React, { Component } from 'react';
import './Search.scss';

class Search extends Component {
 
  render() {
    return (
      <div className="search">
        <input type="search" onChange={this.props.change}></input><button>Cancel</button>
      </div>
    )
  }
}

export default Search;
