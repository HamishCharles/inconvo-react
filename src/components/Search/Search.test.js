import React from 'react';
import Search from './Search';
import renderer from 'react-test-renderer';


test('Renders search component correctly', () => {
  const component = renderer.create(
    <Search change={function(){}} />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});