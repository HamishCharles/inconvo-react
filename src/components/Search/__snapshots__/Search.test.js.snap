// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders search component correctly 1`] = `
<div
  className="search"
>
  <input
    onChange={[Function]}
    type="search"
  />
  <button>
    Cancel
  </button>
</div>
`;
